api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java

```diff
@@ -53,23 +53,26 @@
 
     private Extension ext5;
 
-
     @Before
     public void setUp() throws Exception {
 
+    	// clear up from previous tests
```

Comment:
Please stop sending me your pull requests. I'm getting copied on them by
mistake. Thanks.

On Fri, Feb 12, 2016 at 11:51 AM, James deGraft-Johnson <
notifications@github.com> wrote:

> In
> api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java
> <https://github.com/openmrs/openmrs-module-appframework/pull/14#discussion_r52766355>
> :
>
> >      @Before
> >      public void setUp() throws Exception {
> >
> > +    	// clear up from previous tests
>
> @Wyclif <https://github.com/Wyclif>, DirtiesContext in openmrs 2.0 causes
> the database to be emptied. Tests are not able to proceed because
> authentication is not possible. I agree we need DirtiesContext to maintain
> the tests in appframework but we need to make changes to the platform to
> support DirtiesContext. Particularly, I proposed a change where if
> Authentication in core failed, the database was reloaded.
> I think a more senior dev has to solve the problem with DirtiesContext
> before this ticket can be resolved.
> It is possible that DirtiesContext is not a problem, but this really needs
> to be verified by a senior dev because I am seeing a problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-appframework/pull/14/files#r52766355>
> .
>